src/node_crypto.cc

```diff
@@ -3363,15 +3363,12 @@ bool CipherBase::SetAuthTag(const char* data, unsigned int len) {
 void CipherBase::SetAuthTag(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
 
-  Local<Object> buf = args[0].As<Object>();
-
-  if (!buf->IsObject() || !Buffer::HasInstance(buf))
-    return env->ThrowTypeError("Auth tag must be a Buffer");
+  THROW_AND_RETURN_IF_NOT_BUFFER(args[0], "Auth tag");
```

Comment:
There is a very subtle change in the error message here given that `Buffer` is changed to `buffer`. Unfortunately this means this has to be treated as a semver-major because it's a change in the error message. whee!